paddle/fluid/inference/api/paddle_pass_builder.cc

```diff
@@ -60,6 +61,12 @@ void PaddlePassBuilder::DeletePass(const std::string &pass_type) {
   }
 }
 
+size_t PaddlePassBuilder::GetPassIndex(const std::string &pass_type) {
+  auto iter = std::find(std::begin(passes_), std::end(passes_), pass_type);
+  if (iter == std::end(passes_)) return -1;
+  return std::distance(std::begin(passes_), iter);
+}
+
```

Comment:
@Silv3S  may consider to add such function

Reply:
Thank you. This is exactly what I needed